.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
nit, add new line?

Reply:
Added it, but it doesn't show up in the file changes, so thats interesting

Reply:
the 🛑 symbol shows up if there isn't a new line so it's good now